src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -368,7 +368,7 @@ const TimeEntryForm = props => {
     category = foundProject
       ? foundProject.category.toLowerCase()
       : foundTask?.classification.toLowerCase();
```

Comment:
```suggestion
      : foundTask?.category.toLowerCase();
```
Same as other comment.

Reply:
Done